xonsh/history/base.py

```diff
@@ -103,8 +103,8 @@ def append(self, cmd):
         Parameters
         ----------
         cmd: dict
-            A dict contains informations of a command. It should contains
-            the following keys like ``inp``, ``rtn``, ``ts`` etc.
+            A dict contains information on a command. It should contain
+            the following keys, like ``inp``, ``rtn``, ``ts`` &etc.
```

Comment:
I do believe that a colon or semicolon would be better here.

Also, I learned something new about etc today. I did some internet searching and found out that I've been using it wrong all of my life. Thanks! 😆 

I'll make those changes.